rudderstack_blueprints/trigger_sync.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_SYNC_REFRESH_ERROR = 210
+EXIT_CODE_UNKNOWN_ERROR = 211
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--source-id', dest='source_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def trigger_sync(source_id, access_token):
+    api_headers = {
+        'authorization': f"Bearer {access_token}",
+        'Content-Type': 'application/json'
+    }
+    api_url = "https://api.rudderstack.com/v2/sources"
+    trigger_sync_url = api_url + f"/{source_id}/start"
+    # get response from API
+    try:
+        trigger_sync_response = requests.get(trigger_sync_url, 
+                                           headers=api_headers)
+        # check if successful, if not return error message
```

Comment:
Can't use ok here since that == 200 and some other status codes seem to also indicate success. See other comment.